js/retry-timeout-callback.js

```diff
@@ -218,6 +218,42 @@ var retryTimeoutCallback = (function() {
         editor.addSaveListener(__showStartingBrowser);
     };
 
+    var listenToPlayground = function(editor) {
+        editor.addSaveListener(parseParameters);
+    };
+
+    var __showBrowser = function(editor) {
```

Comment:
Do you have the latest code?   I thought this method was removed in an earlier PR.

Reply:
This must've been from a faulty merge-conflict resolution then.